Kitodo/src/main/java/org/kitodo/services/data/UserGroupService.java

```diff
@@ -354,7 +354,7 @@ private void addBasicUsersRelation(UserGroupDTO userGroupDTO, JsonObject jsonObj
         return convertListIdToDTO(getAllUserGroupIdsByClientIds(clientIdList), this);
     }
 
-    protected List<UserGroup> getAllUserGroupsByClientIds(List<Integer> clientIdList) {
+    public List<UserGroup> getAllUserGroupsByClientIds(List<Integer> clientIdList) {
```

Comment:
Javadoc for public method?

Reply:
@Beacze Can you provide me with a JavaDoc comment? My PR is based on your branch that added this functionality.

Reply:
`/** Get all user roles by list of client ids
@param clientIds list of client ids

@return list of user groups for given client ids */`